application/src/main/java/org/opentripplanner/standalone/config/BuildConfig.java

```diff
@@ -375,6 +378,12 @@ to check in to a flight (2-3 hours for international flights) than to alight and
         """
       )
       .asDouble(DEFAULT_SUBWAY_ACCESS_TIME_MINUTES);
+    submodeMapping = root
+      .of("submodeMapping")
+      .since(V2_8)
+      .summary("CSV file with submode mapping configuration")
+      .description("")
```

Comment:
Either omit the description or put some more information in it. Maybe we should document its structure either here or point to where it's documented.